Improve docs for get-started -> lotus (#899)
filecoin-project/filecoin-docs: docs/get-started/lotus/installation.md

The following instructions are specific to Linux installations. Head to the [macOS](#macos) section if you want to install Lotus on a Mac.

> Users can install Lotus on Linux using the 

Reviewer: [formatting] If you split into a bullet list, the sentence before that should be a complete sentence, and end with `:` (e.g. "Users can install Lotus on Linux using any of these methods:". Or you can leave as a single sentence.

    ./lotus-v1.10.0_linx-amd64.appimage
    ```

> ### Build from source

Reviewer: Thanks for the header level fixes, this is correct logically. Before merging, please check that this doesn't break sidebar formatting in the netlify preview.

Author: Sidebar looks good.

   brew install go bzr jq pkg-config rustup hwloc
   ```

2. Make sure all the packages are installed successfully before you move to next step to build lotus.

### Build and install Lotus

Once all the dependencies are installed, you can build and install the Lotus suite (`lotus`, `lotus-miner`, and `lotus-worker`).

1. Clone the repository:

   ```sh
   git clone https://github.com/filecoin-project/lotus.git
   cd lotus/
   ```

> 2. To join mainnet, checkout the [latest release](https://github.com/filecoin-project/lotus/releases).

Reviewer: ```suggestion
1. Checkout the release for the network you wish to use.

To join mainnet, checkout the [latest release](https://github.com/filecoin-project/lotus/releases). For networks other than mainnet...
```

Also move text from line 315 to the end of the line above. This introduces the idea of different networks first, as you suggested in our chat.

   This is due to a Lotus bug that prevents Lotus from running on a processor without `adx` instruction support, and should be fixed soon.

> 5. Build and install Lotus

Reviewer: Regarding list number, using `1.` for each item is a way to write ordered lists in Markdown. This supports easier re-numbering in the future, if necessary. Let's preserve that since we use it across the repo.

More info: https://www.markdownguide.org/basic-syntax/#ordered-lists

Author: Ok, I will update.

>    Lotus is compiled to operate on a single network,  run one of the following commands to build the lotus node for the specific lotus network.

Reviewer: ```suggestion
   Lotus is compiled to operate on a single network. Run one of the following commands to build lotus for your intended network.
```

A few nit-picks: 
1. Separate the two sentences with a period.
2. Networks are not lotus-specific; you can use any implementation (venus, forest, etc.) to connect to them.
3. Once the lotus application is built, you can use it to kick off multiple nodes. So technically speaking you're not building a lotus node.

   ```sh
   # mainnet
   make clean && make all 
   
   # Or to join a testnet or devnet:
   make clean && make calibnet # Calibration with min 32 GiB sectors
   make clean && make nerpanet # Nerpa with min 512 MiB sectors
   
   sudo make install
   ```

>    Once the installation is finished, use the command down below to ensure lotus is installed successfully for the right network. 

Reviewer: ```suggestion
   Check that lotus is installed successfully for the right network. 
```

Shortened for clarity.

lotus sync wait
```

> While waiting the syncing process, you can also check  status of the syncing process.

Reviewer: ```suggestion
While you wait, you can check the status of the syncing process:
```

Simplify for clarity.

Reviewer: This sections is a duplicate of line 427. Let's keep one, not both. I prefer the placement in line 427 but up to you.

Author: Make sence, I will remove keep line 427 since `lotus sync wait` will also print out the status.

```sh
lotus sync status
```

## Interact with the daemon

The `lotus` command allows you to interact with a _running_ Lotus daemon. The `lotus-miner` and `lotus-worker` commands work in the same way.

> Lotus comes with built-in CLI documentation. Based on different usages, lotus provides different sub-commands for developers. You can also use `help` to check the usage for any sub-command.

Reviewer: ```suggestion
Lotus comes with built-in CLI documentation.
```

These instructions are already really long, and you have nice examples below. I made some suggestions to incorporate this content inside lines 413-423

```sh
lotus
  - chain: Interact with filecoin blockchain
  - client: Make deals, store data, retrieve data
  - wallet: Manage wallet
  - net: Manage P2P Network
  - sync: Inspect or interact with the chain syncer
  ...
  
# Show general help
lotus --help
# Show help for the "client" to make deals, store data, retrieve data
lotus client --help
```

> For example, after your Lotus daemon has been running for a few minutes, use `lotus sync` to check the syncing status of your lotus node .

Reviewer: ```suggestion
For example, after your Lotus daemon has been running for a few minutes, use `lotus sync` to check the sync status of your lotus node.
```

```sh
lotus net sync

> sync status:
...
> 	Target:	[bafy2bzaceaki6bjhe2lxmtyexcff6vh5y5uw4tmbjr3gatwvh5dhaqqb2ykaa] (320791)
> 	Stage: complete
> 	Height: 320791
...
```

> Or use`lotus net` to check the number of other peers that it is connected to in the Filecoin network.

Reviewer: ```suggestion
Or use `lotus net` to check the number of other peers that it is connected to in the Filecoin network.
```
